spec/spec_helper.rb

```diff
@@ -28,7 +28,9 @@
   end
 
   config.after :suite do
-    require "govuk/lint/cli"
-    Govuk::Lint::CLI.new.run([])
+    unless ENV["SKIP_LINT"]
```

Comment:
Don't reckon we need to. This is just to run tests with a bit less noise in dev.